CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Add protocol support for WASM. ([#852](https://github.com/getsentry/relay/pull/852))
 - Add dynamic sampling for transactions. ([#835](https://github.com/getsentry/relay/pull/835))
 - Send network outage metric on healthcheck endpoint hit. ([#856](https://github.com/getsentry/relay/pull/856))
+- Add dynamic sampling for events (errors). ([#883](https://github.com/getsentry/relay/pull/883))
```

Comment:
_nit_: Since this is going out in one release, you can merge the two sampling changelog entries:

```markdown
- Add dynamic sampling for transactions and errors. ([#835](https://github.com/getsentry/relay/pull/835), [#883](https://github.com/getsentry/relay/pull/883))
```

Reply:
ok